src/OrleansRuntime/Placement/ActivationCountPlacementDirector.cs

```diff
@@ -54,8 +52,8 @@ public ActivationCountPlacementDirector(DeploymentLoadPublisher deploymentLoadPu
                     "GlobalConfig.ActivationCountBasedPlacementChooseOutOf is " + globalConfig.ActivationCountBasedPlacementChooseOutOf);
 
             chooseHowMany = globalConfig.ActivationCountBasedPlacementChooseOutOf;
-            this.deploymentLoadPublisher = deploymentLoadPublisher;
-            this.deploymentLoadPublisher.SubscribeToStatisticsChangeEvents(this);
+            deploymentLoadPublisher = deploymentLoadPublisher;
+            deploymentLoadPublisher?.SubscribeToStatisticsChangeEvents(this);
```

Comment:
Isn't it possible to pass in an mock or dummy instance of DeploymentLoadPublisher instead in the afected test? Because it feels that without this, this would not work as expected in a real Silo deployment.
If it's too big a change, then I don't feel that strongly about it anyway